content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -3,28 +3,24 @@ reviewers:
 - bprashanth
 - liggitt
 - thockin
-title: Configure Service Accounts for Pods
+title: Configure ServiceAccounts for Pods
 content_template: templates/task
 weight: 90
 ---
 
 {{% capture overview %}}
-A service account provides an identity for processes that run in a Pod.
+Kubernetes recognises two distinct ways for clients outside the
```

Comment:
hello @sftim . I read through the page. There are a few **minor** nits that could be addressed before merging.
line 12: recognizes 😄 
line 19: spelling `yoy`, recognizes
line 40: capitalize namespace or not?
line 61: the `kubelet`?  enclose in backticks or not?
line 95: possible edit: You can list all ServiceAccount resources in your current namespace with:
line 120: possible edit: You can get a YAML output of ...
line 314: fix the link with forward slash before docs, [projected volume](/docs/concepts/storage/volumes/#projected) type called